src/auth/feature-oauth/Oauth.tsx

```diff
@@ -0,0 +1,23 @@
+import { OauthBadge } from "../ui-oauth-badge";
+import styles from "./Oauth.module.scss";
+import classNames from "classnames/bind";
+import { OAUTH_BADGES } from "./constant";
+
+const cx = classNames.bind(styles);
+
+type OauthProps = {
+  description: string;
+};
+
+export const Oauth = ({ description }: OauthProps) => {
+  return (
+    <div className={cx("container")}>
+      <p className={cx("description")}>{description}</p>
+      <div className={cx("links")}>
+        {OAUTH_BADGES.map((badgeData) => (
+          <OauthBadge {...badgeData} key={badgeData.alt} />
+        ))}
+      </div>
+    </div>
+  );
+};
```

Comment:
```suggestion
export const SocialBadges = ({ description }: OauthProps) => {
  return (
    <div className={cx("container")}>
      <p className={cx("description")}>{description}</p>
      <div className={cx("links")}>
        {OAUTH_BADGES.map((badgeData) => (
          <OauthBadge {...badgeData} key={badgeData.alt} />
        ))}
      </div>
    </div>
  );
};
```

`oAuth`는 인증과 관련된 접근 위임을 위한 개방형 표준으로 불리웁니다 ! 
UI를 표시하는 컴포넌트와는 성격이 조금 다른 것 같아서 제안드립니다 =)